lms/static/js/student_account/views/AccessView.js

```diff
@@ -161,7 +162,8 @@
                         fields: data.fields,
                         model: model,
                         thirdPartyAuth: this.thirdPartyAuth,
-                        platformName: this.platformName
+                        platformName: this.platformName,
+                        registrationFooter: this.registrationFooter,
```

Comment:
This will probably turn into a merge conflict if edX ever updates this collection.

Specifically, because of the extra added comma after `platformName`, git thinks we're changing more than we are. So if edX ever adds another element to the end of the collection, it will also look like they're changing more too. And if git ever suspects both "sides" modified a line at the same time, that's where conflicts arise.

We can avoid this by adding our new line, `registrationFooter`, basically anywhere but at the very end; we could add it as the first element in the dictionary. Then you can remove the added trailing comma from the `platformName` line.